### ISSUE TITLE
How to setup OpenMVG as an external library in Xcode

### ISSUE THREAD

**Issue by cmdavies** [openMVG/openMVG#1299]
Hi everyone,

I have tried to build and install openMVG and the include and lib files live in /usr/local/include and /use/local/lib respectively. 

Just wondering if anyone has got experience of how to setup openMVG as an external library in a Xcode project? 

Thanks a lot in advance for your help!

Chen

**Comment by pmoulon** [comment]
If you project is using cmake, it's easy.
https://github.com/openMVG/openMVG/blob/master/BUILD.md#using-openmvg-as-a-third-party-library-dependency-in-cmake

Else just add those library and some include path to your project and you are ready to go.

**Comment by cmdavies** [comment]
Hi Pierre,

Thanks a lot for your reply.

I have followed the build instruction in your github page and did:
git clone —recursive //http:/github.com/openMVG/openMVG.git
So should be from your current branch?

The openMVG has been built and installed and I can see:
all static libs .a files are in /usr/local/lib
All header files are in /usr/local/include/openMVG and
/usr/local:include/openMVG_dependencies

I tried to add the /usr/local/lib into the Header Search Paths of my
project settings and then include a header file in my code . I have also
added /usr/local/lib in my library search paths, etc. Generally to follow
the steps in the below link (about using openCV in Xcode):

https://blogs.wcode.org/2014/11/howto-setup-xcode-6-1-to-work-with-opencv-libraries/

However when compiling, it fails and tells me that the compiler couldn’t
find the header file. The problem has been detailed in:

https://stackoverflow.com/questions/50010331/xcode-couldnt-find-header-file-although-its-added-into-the-header-search-paths

Any ideas for how to solve this issue? And if my current approach of
setting openMVG up in my Xcode project are correct? I’ve spent the past
several days trying to set openMVG up for my project but start to lose my
will now.

Thanks a lot for your help and insights!
